feat: end_time and duration

shouan2212 opened the pull request.

shouan2212 commented:
• spec/core/test_core/test_manager_spec.rbにupdate_job_recordの期待を追加

shouan2212 commented:
```bash
- Apply hash value shorthand syntax (Ruby 3.1+ feature)
- Use each_value instead of values.each
- Remove unnecessary 'then' in multiline when statements
- Fix layout issues with empty lines

This resolves 23 RuboCop violations detected in static code analysis."
```

nk-ty reviewed (changes requested):
@shouan2212 
Thank you for the fix. I've added review comments, please take a look.

shouan2212 commented on lib/bucky/core/database/test_data_operator.rb:
```diff
@@ -106,11 +106,28 @@ def get_test_case_id(test_suite_id, case_name)
+          puts "DEBUG: Updating job #{job_id} with end_time #{end_time} and duration #{rounded_duration}"
+          return if $debug
+
+          @connector.connect
```
このconnectはいらないです。削除しました。
動作確認：
```bash
| id | start_time          | end_time            | duration | actual_seconds | command_and_option                                  |
                                                                                                                      +----+---------------------+---------------------+----------+----------------+-----------------------------------------------------+
                        | 10 | 2026-01-29 14:23:07 | 2026-01-29 14:23:16 |     9.41 |              9 | bucky run -t e2e -D pc -c bukken_list_new_mansion_1 |

shouan2212 committed:
Add execution duration tracking to jobs

- Add update_job_record method to save end_time and duration
- Update test_manager to call update_job_record after test execution
- Remove debug prints and unnecessary connection calls
- Update CI/CD configuration for deployment
- Add specs for job record updates
